anaconda.spec.in

```diff
@@ -195,6 +195,39 @@ The anaconda-install-env-deps metapackage lists all installation environment dep
 This makes it possible for packages (such as Initial Setup) to depend on the main Anaconda package without
 pulling in all the install time dependencies as well.
 
+%package install-image-deps
```

Comment:
If we want the name a little shorter, it could be `anaconda-install-img-deps`. It nicely complements the `anaconda-install-env-deps` package.